### DISCUSSION TITLE
My IP address (behind NAT) is blocked

### DISCUSSION THREAD

**Discussion by zbycz** [vercel/community#7403] [Help]
### Summary

My ISP is using NAT – so there are lots of users behind 1 IP address.  Recently (~1week) vercel has been blocking all requests to vercel.com or any other vercel deployed site. My IP: 178.255.168.23

When I connect through my cell phone service, everyhing works fine.

### Steps to Reproduce

```
❯ ping vercel.com
PING vercel.com (76.76.21.9): 56 data bytes
Request timeout for icmp_seq 0
```

**Comment by pawlean** [comment]
Hi @zbycz!

We created a debugging tool [here](https://github.com/vercel-support/vercel-connect-debug), can you please run it and send us the output? It'll help us understand what the problem is!

**Reply by AstroboiSteve** [reply]
Hi @pawlean 

I am not OP but am in a very similar situation - my IP address appears to be blocked from all Vercel hosted sites, and my ISP uses CGNAT. I've attached output from the debugging tool.

Thanks
[vercel-debug.txt](https://github.com/user-attachments/files/16154202/vercel-debug.txt)

**Reply by pawlean** [reply]
@AstroboiSteve I will share this internally. I also wanted to share:

If you are using a shared network, like the WiFi at your workplace or university, you might need to ask your network administrator for assistance with unblocking Vercel sites. If you are on a personal network, consider contacting your Internet Service Provider (ISP) directly, as they may respond more promptly to their customers than to external inquiries.

**Reply by zbycz** [reply]
Hi @pawlean , thanks for your answer and for the tool. I already called my provider, and they said they can't do much if it is a ban of our NAT IP adress. 

File is here:
[vercel-debug.txt](https://github.com/user-attachments/files/16193476/vercel-debug.txt)

**Reply by OssianN** [reply]
I have the exact same issue. I'm not sure whether my ISP is using NAT or not, but this is my private WiFi at home.
Please help!

**Reply by pawlean** [reply]
@OssianN Could you try the steps we outlined [here](https://vercel.community/t/resolving-ip-blocking-issues/171)?

**Reply by pawlean** [reply]
@zbycz @AstroboiSteve Thank you for your patience, we're working through these issues. 🙏🏼

**Reply by OssianN** [reply]
Thank you, @pawlean! After having reached out to my ISP and done a `traceroute` to vercel.com they believe the issue is on Vercel's side. They will reach out to see what can be done.

**Reply by AstroboiSteve** [reply]
I ended up getting my ISP to take me off CG-NAT, resulting in a new IP address, and now I can access Vercel services again.
There really should be a better way to have these cases reviewed/resolved in a timely way.

**Reply by pawlean** [reply]
Appreciate all the updates, thank you everyone for your patience and feedback. 🙏🏼

**Reply by zbycz** [reply]
Hi @pawlean , i have unmarked the [answer](https://github.com/orgs/vercel/discussions/7403#discussioncomment-10083957) you chosen, because it is still not resolved in any way. All the websites running on vercel platform are still inaccessible from my ISP and they also believe that the issue is on the vercel side.  (The same as @OssianN said)

Please is there still work being done on your side? thx. 🤞

**Reply by pawlean** [reply]
Apologises, @zbycz I had a few discussions open. We're still investigating your case!

**Comment by pawlean** [comment]
@zbycz Similar to @OssianN, could you execute TCP based traceroute: traceroute -T -p 443 vercel.com? BTW, feel free to drop into our new community (https://vercel.community) It might be easier to DM you directly on there.

**Reply by dvdkon** [reply]
I have the same ISP (and the same problem) as @zbycz, so I ran the command:

<details><summary>Output of sudo traceroute -T -p 443 vercel.com</summary>

```
traceroute to vercel.com (76.76.21.142), 30 hops max, 60 byte packets
 1  OpenWrt.lan (10.0.1.1)  3.964 ms  0.407 ms  0.390 ms
 2  * * *
 3  100.100.115.254 (100.100.115.254)  10.303 ms  10.406 ms  10.274 ms
 4  cust-starnet.supernetwork.cz (88.86.97.137)  16.102 ms cust-starnet.supernetwork (88.86.97.209)  7.809 ms  10.651 ms
 5  151.148.13.108 (151.148.13.108)  16.209 ms 151.148.13.110 (151.148.13.110)  10.611 ms 151.148.13.108 (151.148.13.108)  7.507 ms
 6  150.222.81.29 (150.222.81.29)  31.857 ms 150.222.81.13 (150.222.81.13)  31.516 ms 150.222.81.39 (150.222.81.39)  31.481 ms
 7  * * *
 8  * * *
 9  * * *
10  * * *
11  * * *
12  * * *
13  * * *
14  * * *
15  * * *
16  * * *
17  * * *
18  * * *
19  * * *
20  * * *
21  * * *
22  * * *
23  * * *
24  * * *
25  * * *
26  * * *
27  * * *
28  * * *
29  * * *
30  * * *
```

</details>

**Reply by pawlean** [reply]
Hi @dvdkon!

Can you head over to the vercel.community and post there? It'd help me manage this better :)

**Reply by zbycz** [reply]
Posted here https://vercel.community/t/my-ip-address-behind-nat-is-blocked/411 - thanks @pawlean  🤞 

@dvdkon - as we don't share any hops, i guess we may be on different part of network..?

[a comment by an automated account is not shown]
